elao.app.docker/README.md

```diff
@@ -474,6 +474,57 @@ test.jest@integration:
 
 ```
 
+## Webpack
+
+Open websocket port in `.manala.yaml` (`[XXX]` being project's ports first three numbers):
+```yaml
+system:
+    ...
+    docker:
+        ...
+        services:
+            ...
+            # App
+            app:
+                ports:
+                  # Webpack
+                  - [XXX]88:8080
+```
+
+Ensure related webpack dev server options are defined in `webpack.config.js`:
+```javascript
+...
+.configureDevServerOptions(options => {
+    // See: https://symfony.com/doc/current/frontend/encore/dev-server.html#cors-issues
+    options.allowedHosts = 'all';
+    options.port = 'auto';
+})
+```
+
+Define following targets in `Makefile` (`[XXX]` being project's ports first three numbers):
+```makefile
+#########
+# Watch #
+#########
+
+## Build assets and watch for changes
+watch:
```

Comment:
Since we have a working dev server, I don't see the point of keeping the watch target actually